### PR TITLE
Add 0.5.0 changelog and bump version

### PR DESCRIPTION
**Pull request by IoTMOD**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,58 @@
...
+Instructions on how to update to this version will be posted in due time.  
+The old HORNET versions won't be functional within the mainnet anymore!  
+
+### Added 
+
+    - White-Flag
+    - Conflicting badge to the transaction explorer
+    - Weighted uniform random tipselection 
+    - Adaptive heaviest branch tipselection
+    - COO heaviest branch tipselection
```
Comment (alexsporn) [inline CHANGELOG.md]: Duplicate with the line before

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,58 @@
...
+    - Neighbors count to heartbeat message
+    - LMI and neighbor counts to dashboard
+    - getTipInfo API call
+
+### Changed
+
+    - Return false for conflicting tx in the getInclusionStates web api call
+    - Request tx from all neighbors that could have the data
+    - Bump protocol feature set for whiteflag
+    - Reconnect websocket on failue
```
Comment (alexsporn) [inline CHANGELOG.md]: Typo

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,58 @@
...
+### Fixed
+
+    - Missing byte to trytes conversion in some error messages
+    - Make code more testable
+    - Adding autopeered neighbors as static neighbors
+    - CTPS calculation
+    - Revalidation 
+    - Pruning
+    - High dashboard traffic
+    - Misc page not shown if spammer is enabled
```
Comment (alexsporn) [inline CHANGELOG.md]: This was a temporary bug, not in 0.4.2

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,58 @@
...
+    - Graph plugin
+    - Monitor plugin
+    - Legacy gossip protocol
+    - Genesis tx special case
+
+### Config file changes
+
+Please use the new config.json and transfer values from your current config.json over to the new one, as a lot of keys have changed or got removed (instead of mutating your current one).
+
 ## [0.5.0-rc13] - 14.08.2020
```
Comment (alexsporn) [inline CHANGELOG.md]: Remove all RCs?

Comment (IoTMOD) [reply CHANGELOG.md]: As they are partly incomplete, this may be a good idea, yes :+1:

**Review by alexsporn — APPROVED** [review]

**Review by muXxer — APPROVED** [review]

**Pull request merged**